### ISSUE TITLE
Bug: Breaking change in new release of Starlette

### ISSUE THREAD

**Issue by miguescri** [tiangolo/fastapi#2342]
Yesterday, [Starlette 0.14.1](https://github.com/encode/starlette/releases/tag/0.14.1) was released, which removes `UJSONResponse`. As a result, https://github.com/tiangolo/fastapi/blob/b53c443a06f898189e0f458dd1d7cbe55742192b/fastapi/responses.py#L10 breaks on start-up.

**Comment by ycd** [comment]
Here is the related PR #2335

**Comment by miguescri** [comment]
Thank you for the pointer @ycd .

I'll keep the issue open until the PR merges, for reference for users whose servers break (like mine did).

**Comment by mezhaka** [comment]
What's the smartest thing to do here?
I use conda to create my environment and I have currently pinned `starlette==0.13.8`, which seems to help.

**Comment by miguescri** [comment]
Yes, until #2335 merges, the Starlette version needs to be pinned to <0.14.1 if you have it in your requirements.

@Kludex , for most people that will do. But it is worth noting that may not be an option if you directly use some of Startlette's elements and need to have an updated version. Keep in mind that right now FastAPI installs by default Starlette 0.13.6, which is 4 releases behind master.

**Comment by mezhaka** [comment]
@Kludex @miguescri I did not have explicit Starlette in my environment and this way the app was broken.  As I have added it explicitly -- it runs.

**Comment by Mause** [comment]
Which package manager are you using? If you don't add an explicit dependency version, it should automatically install a compatible version

**Comment by miguescri** [comment]
@mezhaka are you using a virtual env or a system wide installation? And are you using a Python package manager or you installed some of the Python packages from your distro's repositories?

**Comment by mezhaka** [comment]
@Mause @miguescri I use conda.  I have an environment.yml file which looks something like this (I've removed some packages):
```yml
name: myapp
channels:
  - conda-forge
  - mycompany
dependencies:
  - python~=3.7
  - pymongo~=3.10
  - gunicorn~=20.0
  - uvicorn~=0.11
  - fastapi~=0.60
  - pydantic~=1.6
```
I then create environments with `conda env create -f env.yml`, activate it `conda env activate myapp`, and start my python web app.

**Comment by ycd** [comment]
Btw, i'm still not sure why this error occurs, because in our `pyproject.toml`

https://github.com/tiangolo/fastapi/blob/52dd5924d787894bf424fb24a5471089f477485f/pyproject.toml#L35

Starlette pinned to the `0.13.6`, so it shouldn't be installing the `0.14.1`.

**Comment by maresb** [comment]
I also ran into problems with my Conda environment. FastAPI pins `starlette-base` to `0.13.6` but `starlette` itself is unpinned.

I reported it to the feedstock here:
https://github.com/conda-forge/fastapi-feedstock/issues/43

**Comment by miguescri** [comment]
#2335 has been merged.

**Comment by tiangolo** [comment]
Thanks for the help here everyone! 👏  🙇 

Thanks for reporting back and closing the issue @miguescri  👍

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
